src/sentry/rules/conditions/event_attribute.py

```diff
@@ -184,6 +186,23 @@ def _get_attribute_values(self, event: GroupEvent, attr: str) -> Sequence[str]:
                 if device is None:
                     device = []
                 return [device.get(path[1])]
+
+        elif path[0] == "app":
+            if path[1] in ("in_foreground"):
+                contexts = event.data["contexts"]
+                response = contexts.get("app")
+                if response is None:
+                    response = {}
+                return [response.get(path[1])]
+
+            return []
+        # TODO: review this
+        elif path[0] == "thread":
+            if path[1] not in ("main"):
+                return []
+
+            return [getattr(e, path[1]) for e in event.interfaces["threads"].values]
```

Comment:
I don't know if `event.interfaces["threads"]` works, I've used `event.interfaces["exception"]` as an example, `threads` are not being used as of now.

Reply:
Will make another PR